src/components/branding/Branding.tsx

```diff
@@ -13,14 +13,15 @@ export interface BrandingProps extends HeadingProps {
   logo?: LogoProps;
   heading: string;
   hasLogoHeadingDivider?: boolean;
+  size?: string;
 }
 
 export const Branding: FunctionComponent<BrandingProps> = (props) => {
   const {
     logo,
     heading,
     hasLogoHeadingDivider = true,
-    size = ["sm", "md"],
+    size = props ? props?.size : ["sm", "md"],
     ...headingProps
   } = props;
```

Comment:
You can revert the change above and initialise the props i.e.  } = props || {};

Reply:
Updated